README.md

```diff
@@ -133,7 +133,7 @@ Rack::Attack.blocklist_ip("1.2.0.0/16")
 
 #### `blocklist(name, &block)`
 
-Name your custom blocklist and make your ruby-block argument returna a truthy value if you want the request to be blocked, and falsy otherwise.
+Name your custom blocklist and make your ruby-block argument returns a truthy value if you want the request to be blocked, and falsy otherwise.
```

Comment:
Hi @sdnorm,

Thanks for spotting the typo and opening the pull request!

Should it be "return" instead of "returns"?

Reply:
Ahh yes, it should be. I can fix it and resubmit my pull request.

Reply:
Thanks!

By the way, if you have any more feedback about the README let me know. There's been some updates to the README recently to make it more readable and easier to understand. So any feedback in that category, either by commenting here and/or opening and issue is greatly appreciated.

:+1: